cmd/rook/ceph/cleanup.go

```diff
@@ -67,7 +67,7 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 	}
 
 	namespace := os.Getenv(k8sutil.PodNamespaceEnvVar)
-	clusterInfo := client.AdminClusterInfo(namespace)
+	clusterInfo := client.AdminClusterInfo(namespace, "")
```

Comment:
Not sure about this case, the empty string may be fine here.

Reply:
By setting it to empty, if this code path tries to use the cephcluster CR name it will throw an exception and immediately crash and help us realize it is required. I don't believe it is required in this scenario though.